lib/mix/lib/mix/tasks/test.ex

```diff
@@ -549,7 +549,7 @@ defmodule Mix.Tasks.Test do
   defp failed_opts(opts) do
     if opts[:failed] do
       if opts[:stale] do
-        Mix.raise("Combining `--failed` and `--stale` is not supported.")
+        Mix.raise("Combining --failed and --stale is not supported.")
```

Comment:
IIRC we use double quotes for commands. I think the same should apply for command arguments

Reply:
I saw the double quotes for commands, but I think command arguments are printed without them. For example:

https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/tasks/xref.ex#L642
https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/tasks/xref.ex#L559
https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/tasks/run.ex#L104-L107

Reply:
the problem with not using quotes when it is not at the end of the line is that command options can take arguments, so it can be ambiguos.

See https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/tasks/run.ex#L106

it should read 

    and "--require VAL" for requiring

to avoid ambiguity and for clarity

Reply:
I guess we can be conservative and use it when needed

Reply:
But, I would always use double-quotes when a command option with an argument are present